src/Tools/dotnet-user-jwts/src/Helpers/DevJwtCliHelpers.cs

```diff
@@ -14,13 +14,13 @@ namespace Microsoft.AspNetCore.Authentication.JwtBearer.Tools;
 
 internal static class DevJwtCliHelpers
 {
-    public static string GetOrSetUserSecretsId(IReporter reporter, string projectFilePath)
+    public static string GetOrSetUserSecretsId(string projectFilePath)
     {
-        var resolver = new ProjectIdResolver(reporter, projectFilePath);
+        var resolver = new ProjectIdResolver(NullReporter.Singleton, projectFilePath);
         var id = resolver.Resolve(projectFilePath, configuration: null);
```

Comment:
Why not pass in a higher configuration level by default and then we can have an option like user-secrets to have a different config level to get verbose logs:
https://source.dot.net/#dotnet-user-secrets/CommandLineOptions.cs,39

Reply:
We don't currently have a verbose flag on the tool, but even if we did I think we'd end up passing either a `ConsoleReporter` or `NullReporter` here depending on whether or not that verbose flag is set.